app/build.gradle

```diff
@@ -7,6 +7,7 @@ apply from: './jacoco.gradle'
 android {
     lintOptions {
         warning 'InvalidPackage'
+        abortOnError false
```

Comment:
> use resolution strategy like multidex

thats not working for this. I think we have these two options:
1. change support library version to `25.3.1` from `25.4.0`
2. disable lint from aborting build on error